### ISSUE TITLE
Remove the "0" when the marcher is on the line

### ISSUE THREAD

**Issue by AlexDumo** [OpenMarch/OpenMarch#271]
This is an easy fix, just add an `=== 0` check to display the number.

What it does now -> `0 On 30 S2`
What it should do -> `On 30 S3`

**Comment by Abhishek022001** [comment]
can i work this issue?

**Comment by AlexDumo** [comment]
Yeah sure, should be a couple lines of code

**Comment by AlexDumo** [comment]
I don't think this is super necessary with how the shape editor looks now. The 0 looks fine

![Image](https://github.com/user-attachments/assets/c7e5a33e-5f28-4e13-9345-ccceac448f81)
